submissions/dashakim/popup_css_html/index.html

```diff
@@ -0,0 +1,173 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>HTML & CSS practice</title>
+    <link rel="stylesheet" href="css/main.css">
+</head>
+
+<body>
+    <header class="site-header">
+        <nav class="nav">
+            <ul class="nav__list">
+                <li class="nav__item">
+                    <a href="#" class="nav__link">Mail</a>
+                </li>
+                <li class="nav__item">
+                    <a href="#" class="nav__link">Images</a>
+                </li>
+                <li class="nav__item">
+                    <input type="checkbox" id="checkboxApps" class="popup-checkbox">
+                    <label for="checkboxApps" class="popup__label">
+                        <img src="src/appsdark.png" class="icon-apps" alt="apps" tabindex="0">
```

Comment:
`input type="checkbox"` already an interactive element. Not necessary to use `tabindex`.
In your case better to  CSS way: 
```CSS
.popup-checkbox:focus + .popup__label {
// styles for focus state
}
```
Use `tabindex` in case you need to make an interactive element from the regular noninteractive element.